.semgrepignore

```diff
@@ -19,6 +19,9 @@ test/
 tests/
 *_test.go
 
+# false positive
+testutil/network/network.go
```

Comment:
Can you provide context for this? AFAICT tests were passing in the previous commit
I'd prefer to not ignore tests.

Reply:
https://github.com/cerc-io/laconicd/runs/12015779774
Weirdly, this check fails only sometimes.

Reply:
#104
please change line 22 to `# false positive; TODO: https://github.com/cerc-io/laconicd/issues/104`
then feel free to merge